src/goPackages.ts

```diff
@@ -278,9 +278,13 @@ export function getNonVendorPackages(folderPath: string): Promise<string[]> {
 // see: https://golang.org/s/go14internal
 function isAllowToImportPackage(toDirPath: string, currentWorkspace: string, pkgPath: string) {
 	let internalPkgFound = pkgPath.match(/\/internal\/|\/internal$/);
+	if (internalPkgFound) {
```

Comment:
shouldnt this be `if(!internalPkgFound)`?

Your case is that `pkgPath` has a value like `internal/nettrace` for which `internalPkgFound` would be null at this point.

Reply:
Thank for the correct.. will fix soon